dev/build/windows/configure_profile.sh

```diff
@@ -14,30 +14,31 @@
 rcfile=~/.bash_profile
 donefile=~/.bash_profile.upated
 
+exec >> $rcfile
+
 if [ ! -f $donefile ] ; then
 
     echo >> $rcfile
```

Comment:
This redirection can be removed (this line just adds a newline to rcfile).

Reply:
Done. I added a `touch $rcfile` at the beginning.